cli.sh

```diff
@@ -96,20 +96,31 @@ set_main_variables() {
 
 #####################################################
 # Checks if the current directory is
-#   a Git repository or not.
-#
+#   a Git repository or not and sets
+#   CURRENT_GIT_DIR_IS_BARE to `false` or `true`
+
 # Returns:
 #   0 if it is likely a Git repository,
 #   1 otherwise
 #####################################################
 is_running_in_git_repo_root() {
-    if ! git status >/dev/null 2>&1; then
-        return 1
-    fi
-
+    CURRENT_GIT_DIR_IS_BARE=$(git rev-parse --is-bare-repository)
```

Comment:
This should be just `git rev-parse || return 1` with redirecting output to devnull. We want to answer here whether we're in a git repo or not, it shouldn't care about bare or not, and should also not have side effects of setting other variables.
Not sure what's `[ -d "${CURRENT_GIT_DIR}" ] || return 1` doing right now, maybe that needs to go now?